File: PRAK605-2310817220016-NailaHanifah.c
#include <stdio.h>

void multiplyMatrices(int n, int A[n][n], int B[n][n], int result[n][n]) {
    for (int i = 0; i < n; ++i) {
        for (int j = 0; j < n; ++j) {
            result[i][j] = 0;
            for (int k = 0; k < n; ++k) {
                result[i][j] += A[i][k] * B[k][j];
            }
        }
    }
}

void printMatrix(int n, int matrix[n][n]) {
    for (int i = 0; i < n; ++i) {
        for (int j = 0; j < n; ++j) {
            printf("%d ", matrix[i][j]);
        }
        printf("\n");
    }
}

int main() {
    int n;

    scanf("%d", &n);

    int A[n][n], B[n][n], result[n][n];

    printf("Matriks A\n");
    for (int i = 0; i < n; ++i) {
        for (int j = 0; j < n; ++j) {
            scanf("%d", &A[i][j]);
        }
    }
    printf("Matriks B\n");
    for (int i = 0; i < n; ++i) {
        for (int j = 0; j < n; ++j) {
            scanf("%d", &B[i][j]);
        }
    }

    multiplyMatrices(n, A, B, result);

    printf("Matriks AXB\n");
    printMatrix(n, result);

    return 0;
}

File: PRAK605-2310817220016-NailaHanifah.py
def multiply_matrices(n, A, B):
    result = [[0 for _ in range(n)] for _ in range(n)]

    for i in range(n):
        for j in range(n):
            for k in range(n):
                result[i][j] += A[i][k] * B[k][j]

    return result

def print_matrix(matrix):
    for row in matrix:
        print(" ".join(map(str, row)))

def main():
    n = int(input())
    print("Matriks A")
    A = [list(map(int, input().split())) for _ in range(n)]
    print("Matriks B")
    B = [list(map(int, input().split())) for _ in range(n)]

    result = multiply_matrices(n, A, B)

    print("Matriks AXB")
    print_matrix(result)

if __name__ == "__main__":
    main()